priv/swagger/swagger-initializer.js

```diff
@@ -3,7 +3,7 @@ window.onload = function() {
 
   // the following lines will be replaced by docker/configurator, when it runs in a docker-container
   window.ui = SwaggerUIBundle({
-    url: window.location.origin + "/api-docs/swagger.json",
+    url: window.location.toString().replace("/index.html", "/swagger.json"),
```

Comment:
Could you explain this change?

Also, I can't remember, but are we testing this somewhat?

You might want to look into `up_swagger.sh` because there's logic for this file, there.

Reply:
I suppose here provided trying dynamically change `api-docs` but... this will not works as expected. Because specific window location will return specific value... Need find other approach.

Reply:
> Also, I can't remember, but are we testing this somewhat?

Based on CI results - no.

Reply:
What if.. during build this project the value `/api-docs/` will be replaced based on config values? I suppose it can be done in rebar.config level as well.

Reply:
yeah… env variable or macro would be great for that.